docs/source/new-project.rst

```diff
@@ -1,72 +0,0 @@
-Quick Start
```

Comment:
shall it be part of TOC them?

Reply:
ok. I added it there for now. I think it got removed at some point, not sure if the introduction guide was supposed to replace it.

Reply:
@williamFalcon where to place it?